docs/modules/migrate/pages/dmt-connection-config.adoc

```diff
@@ -0,0 +1,169 @@
+= Configure Connections
+:description: The migration cluster acts a client for both source and target clusters. You must configure the connections to allow communication between the migration cluster and both the source and target clusters. 
+
+{description}
+
+If the connections are not correctly configured, the migration fails.
+
+== Source Cluster Connection
+
+To complete the configuration, you need the following information from your source cluster:
+
+* The cluster name
+* The cluster member addresses
+* The data structures to migrate
+
+=== Update Cluster Information
+
+To update the cluster information, complete the following steps:
+
+. Navigate to the folder containing the extracted DMT package
+. Open the _migration_config/source/hazelcast-client.yaml_ file in your favorite editor
++
+NOTE: The _hazelcast-client.yaml_ file is a Hazelcast client configuration file, which can include any supported configuration.
++
+. Update the `cluster-name` field to match the name of your source cluster
+. Update the `cluster-members` field to match the addresses of the cluster members
+. Save the file
+
+=== Update Data Structures
+
+To update the data structure information, complete the following steps:
+
+. Navigate to the folder containing the extracted DMT package
+. Open the _migration_config/data/imap_names.txt_ and/or the _migration_config/data/replicated_map_names.txt_ file in your favorite editor
+. Update the file content to match the names of your maps
++
+If you have multiple data structures use a new line for each map name, or use wildcards to select multiple data structures using a single entry. For further information on using wildcards, see the xref:configuration:using-wildcards.adoc[Using Wildcards] topic.
+
+. Save the files
+
+By default, Replicated Map metadata is migrated. However, you must add the custom JAR to your class path and restart the cluster before starting the migration. If this is not done, the Replicated map metadata is not migrated. For further information on adding the custom JAR to your classpath, refer to the https://docs.oracle.com/javase/6/docs/technotes/tools/windows/classpath.html[Oracle documentation,window=_blank].
```

Comment:
For ReplicatedMap with metadata, we should add a note. "It's not possible to migrate replicated map metadata to target versions below 5.3.7." This kind of limitation does not exist for CP. They can be migrated to a target cluster with version 5.2.z for instance.

Reply:
Another thing, we should clarify that the custom jar is for the source cluster and not the target cluster. Also we can give a link to the new page I added in this PR https://github.com/hazelcast/hz-docs/pull/1042